vip-jetpack/connection-pilot/class-jetpack-connection-pilot.php

```diff
@@ -149,12 +149,20 @@ public function reconnect() {
 		if ( true === $connection_attempt ) {
 			if ( ! empty( $this->last_heartbeat['cache_site_id'] ) && (int) \Jetpack_Options::get_option( 'id' ) !== (int) $this->last_heartbeat['cache_site_id'] ) {
 				$this->send_alert( 'Alert: Jetpack was automatically reconnected, but the connection may have changed cache sites. Needs manual inspection.' );
-
 				return;
 			}
 
 			$this->send_alert( 'Jetpack was successfully (re)connected!' );
 
+			$vaultpress_connection_attempt = Connection_Pilot\Controls::connect_vaultpress();
+
+			if ( is_wp_error( $vaultpress_connection_attempt ) ) {
```

Comment:
Maybe log/alert the error? Or does that get logged already ?

Reply:
We need to rework how logging/alerting in Connection Pilot. Currently, we simply send Slack alerts like this when something succeeds or not. 

You are right that in this specific case we could be logging way more information. I'd rather do it like this in the PR, since everything else in CP works similarly. After that, we can rethink how we want to do it (it should be a combination of Slack and Logstash).

WDYT @WPprodigy

Reply:
OK sounds good. Otherwise looks good. So I will approve this, and feel free to address it in later PR.